starfish/image/_stack.py

```diff
@@ -287,7 +288,14 @@ def show_stack(
         data, remaining_inds = self.get_slice(indices)
 
         # identify the dimensionality of data with all dimensions other than x, y linearized
-        n = numpy.dot(*data.shape[:-2])
+        if len(data.shape) == 3:
+            n = data.shape[0]
+        elif len(data.shape) > 3:
+            n = numpy.dot(*data.shape[:-2])
```

Comment:
generically, can't this be replaced by:
`n = numpy.multiply(*data.shape[:2])`

Should handle all the possible dimension shapes.  Plus, don't we always expect the full Z,CH,HYB set for starfish?

Reply:
(1) Yea, I think we could replace dot with multiply.  

(2) No, we don't expect a 5d tensor. The user has the chance to select what they want to visualize in this case (output of `ImageStack.get_slice`). I made this fix because I realized that the original method 

Some examples: 
- all sections (ch, hyb, z) (5d tensor)
- just the tiles for a specific z-layer, across channels and hybs (4d tensor)
- just the tiles for a specific channel and hyb, but across z (3d tensor)

... I suppose it could also take a 2d tensor, but then this function is overkill (you can just use `imshow`)

Reply:
I think `np.product` is the correct choice, not multiply or dot.

Reply:
Good call, no -- we don't. Sorry, hadn't thought the change all the way through.